include/seqan3/alphabet/quality/phred63.hpp

```diff
@@ -0,0 +1,185 @@
+// ARE DISCLAIMED. IN NO EVENT SHALL KNUT REINERT OR THE FU BERLIN BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+// LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// ============================================================================
+// Author: Marie Hoffmann <marie.hoffmann AT fu-berlin.de>
+// ============================================================================
+
+#pragma once
+
+#include <cassert>
+
+namespace seqan3
+{
+
+/*!
+ * The phred63 alphabet structure represents the zero-based phred score range
+ * [0..62], mapped to the ascii-ordered range ['@' .. '~'] with
+ * 0=unused, 1=unused, 2=Read Segment Quality Control Indicator. It represents
+ * the Illumina 1.3+ standard score.
+*/
+struct phred63
+{
+    //!\publicsection
+    /*!\name Member types
+    * \{
+    */
+    //!\brief The 0-based integer representation of a quality score.
+    //!\hideinitializer
+    using rank_type = uint8_t;
+
+    //!\brief The integer representation of a quality score.
+    //!\hideinitializer
+    using phred_type = uint8_t;
+
+    //!\brief The '@'-based character representation of a quality score.
+    //!\hideinitializer
+    using char_type = char;
+    //!\}
+
+    /*!\name Member variables.
+    * \{
+    */
+    //!\brief The internal 0-based rank value.
+    rank_type _value;
+
+    //!\brief The projection offset between char and rank quality score representation.
+    //!\hideinitializer
+    static constexpr char_type offset_char{'@'};
+    //!\}
+
+    //!\brief Value assignment with implicit compatibility to inner type.
+    constexpr phred63 & operator =(phred_type const c)
+    {
+        assert(c < value_size);
+        _value = c;
+        return *this;
+    }
+
+    /*!\name Comparison operators.
+    * \{
+    */
+    constexpr bool operator==(const phred63 & rhs) const
+    {
+        return this->_value == rhs._value;
+    }
+
+    constexpr bool operator!=(const phred63 & rhs) const
+    {
+        return this->_value != rhs._value;
+    }
+
+    constexpr bool operator<(const phred63 & rhs) const
+    {
+        return this->_value < rhs._value;
+    }
+
+    constexpr bool operator>(const phred63 & rhs) const
+    {
+        return this->_value > rhs._value;
+    }
+
+    constexpr bool operator<=(const phred63 & rhs) const
+    {
+        return this->_value <= rhs._value;
+    }
+
+    constexpr bool operator>=(const phred63 & rhs) const
+    {
+        return this->_value >= rhs._value;
+    }
+    //!\}
+
+    /*!\name Conversion and explicit conversion functions.
```

Comment:
Please use the group names that the other alphabets are also using, i.e. "Read functions" for `to_*` and "Write functions" for `assign_*`